greylist_retry.py

```
@@ -0,0 +1,49 @@
+                await upsert_greylist(email, mx_host, RETRY_DELAY_BASE * (2 ** tries), tries)
+        await asyncio.sleep(120)
+
+async def probe_email(mx_host: str, email: str) -> bool:
+    try:
+        smtp = SMTP(hostname=mx_host, port=25, timeout=5)
+        await smtp.connect()
+        await smtp.helo("verify.local")
+        await smtp.mail("")
+        code, _ = await smtp.rcpt(email)
+        await smtp.quit()
+        return code == SMTPStatus.completed
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)